shellcheck.yml

```diff
@@ -0,0 +1,43 @@
+name: Shellcheck Lint
+
+on:
+  push:
+    paths:
+      # Run workflow on push to file pihole_adlist_tool
+      - pihole_adlist_tool
+
+  pull_request:
+    branches:
+      # Run workflow on pull request to master
+      - master
```

Comment:
I would remove that. I'm happy if it would run on all paths and all branches.
```suggestion

```

Reply:
This won't work. We need this instead:
```
on:
  push:
    branches:
      # Run workflow on every push
      - '**'        # matches every branch

  pull_request:
    branches:
      # Run workflow on every push
      - '**'        # matches every branch
```